src/sentry/api/endpoints/auth_index.py

```diff
@@ -67,9 +67,9 @@ def _reauthenticate_with_sso(request, org_id):
 
     @staticmethod
     def _verify_user_via_inputs(validator, request):
-        # See if we have a u2f challenge/response
         if "challenge" in validator.validated_data and "response" in validator.validated_data:
             try:
+                metrics.incr("2fa.login_attempt", sample_rate=1.0, skip_internal=False)
```

Comment:
After talking with @leedongwei, the SLOs to track password and 2FA will track that we have a certain number of 2FA and password logins over a certain time period